### PR TITLE
feat: support multivers x

### PR DESCRIPTION
**Pull request by Wozacosta**
### 📝 Description

_Replace this text by a clear and concise description of what this pull request is about and why it is needed._

### ❓ Context

- **Linked resource(s)**: [] 

### 🚀 Expectations to reach

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-9879]: https://ledgerhq.atlassian.net/browse/LIVE-9879?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Justkant — COMMENTED** [review]
LGTM just some comments

File: src/hooks/useProposal/util.ts [hunk 1]
```diff
@@ -81,7 +81,10 @@ export const formatAccountsByChain = (
         isRequired: families.some(
           (family) => family.required && family.chains?.includes(chain)
         ),
-        accounts: accounts.filter((acc) => acc.currency === formatedChain),
+        accounts: accounts.filter((acc) => {
+          if (formatedChain === 'mvx:1' && acc.currency === 'elrond') return true;
```
[the remark this answers is not in the document]
Comment (Wozacosta) [reply src/hooks/useProposal/util.ts]: yup it's better in `getCurrencyByChainId` 

done in this commit https://github.com/LedgerHQ/wallet-connect-live-app/pull/159/commits/d8bb42f20fc79e94c178adda5afb1665d0cc6b8a

File: src/components/screens/SessionDetail.tsx [hunk 2]
```diff
@@ -56,11 +56,16 @@ const getAccountsFromAddresses = (addresses: string[], accounts: Account[]) => {
   addresses.forEach((addr) => {
     const addrSplitted = addr.split(":");
     const chain = getCurrencyByChainId(`${addrSplitted[0]}:${addrSplitted[1]}`);
+    let chainInLedgerLive = chain
 
-    const existingEntry = accountsByChain.get(chain);
+    if (chain === "mvx:1") {
```
Comment (Wozacosta) [inline src/components/screens/SessionDetail.tsx:61]: could use startsWith

File: src/utils/helper.util.ts [hunk 3]
```diff
@@ -50,6 +60,9 @@ export const getNamespace = (chain: string) =>
   SUPPORTED_NETWORK[chain]?.namespace ?? chain;
 
 export const getCurrencyByChainId = (chainId: string) => {
+  if (chainId === 'mvx:1') {
+    return 'elrond';
+  }
```
Comment (Wozacosta) [inline src/utils/helper.util.ts:65]: check if removal ok?

**Review by Justkant — APPROVED** [review]

**Pull request merged**